ONSdigital/eq-survey-runner
# eq-176 respondant page layout

This PR adds the basic responsive page layout as designed in the [prototype](http://uxresearch.eq.ons.digital/V1/index.html).

<img width="906" alt="screen shot 2016-02-17 at 15 34 16" src="https://cloud.githubusercontent.com/assets/930398/13114593/053820d6-d58c-11e5-8760-8a092875a814.png">
## Changes
- added a route at `/mci/`.
- added page template.
- styled components used in creating layout include header information box.
## How to test
- visit `/mci/` and verify page looks like the screenshot above.
- compare with [prototype](http://uxresearch.eq.ons.digital/V1/index.html).
#### Notes
- "Back to Survey Management" is not included yet.
## Who can test

Anyone.
